src/api/goods.ts

```diff
@@ -10,10 +10,15 @@ export function getAll(): Promise<Good[]> {
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => goods
+      .sort((a, b) => a.name.localeCompare(b.name))
+      .slice(0, 5));
 };
 
 export const getRedGoods = () => {
   return getAll()
-    .then(goods => goods); // get only red
+  // eslint-disable-next-line
+    .then(goods => goods
+      // eslint-disable-next-line
+      .filter(goods => goods.color === 'red'));
```

Comment:
i guess parenthesis would satisfy eslint
```suggestion
    .then((goods) => goods
      .filter((goods) => goods.color === 'red'));
```